daemon/exec.go

```diff
@@ -17,34 +17,34 @@ import (
 	"github.com/docker/docker/pkg/pools"
 	"github.com/docker/docker/pkg/signal"
 	"github.com/docker/docker/pkg/term"
-	"github.com/opencontainers/runtime-spec/specs-go"
+	specs "github.com/opencontainers/runtime-spec/specs-go"
 	"github.com/pkg/errors"
 	"github.com/sirupsen/logrus"
 )
 
 // Seconds to wait after sending TERM before trying KILL
 const termProcessTimeout = 10 * time.Second
 
-func (d *Daemon) registerExecCommand(container *container.Container, config *exec.Config) {
```

Comment:
why rename `d` to `daemon` ?

Reply:
can do the reverse as well; rename it to `d` everywhere. Point was that in the same package, the receiver was sometimes named `daemon`, sometimes `d`, which linters in my IDE kept complaining about.

Reply:
Yeah, it's a bit of a split; 

https://github.com/golang/go/wiki/CodeReviewComments#receiver-names

> The name of a method's receiver should be a reflection of its identity; often a one or two letter abbreviation of its type suffices (such as "c" or "cl" for "Client"). Don't use generic names such as "me", "this" or "self", identifiers typical of object-oriented languages that gives the method a special meaning. In Go, the receiver of a method is just another parameter and therefore, should be named accordingly. The name need not be as descriptive as that of a method argument, as its role is obvious and serves no documentary purpose. It can be very short as it will appear on almost every line of every method of the type; familiarity admits brevity. **Be consistent, too: if you call the receiver "c" in one method, don't call it "cl" in another**.


From the _consistency_ perspective, `daemon` dwarfs `d` in this package, so I thought it'd make sense to go for consistency

- 331 occurrences of `func (daemon *Daemon)`
-  16 occurrences of `func (d *Daemon)`